src/app/overview/page.tsx

```diff
@@ -0,0 +1,127 @@
+            開催期間中は、学生団体や地域の皆さまと協力してつくりあげた展示や体験型企画、ステージイベントなどが行われます。
+          </Text>
+          <Text>
+            普段は落ち着いた雰囲気の赤羽台キャンパスが、この期間は活気にあふれたにぎやかな場所となります。
+          </Text>
+        </Container>
+        <Container className={styles.noPadding}>
+          <Text>
+            この大学祭が、あなたと新しい何かを繋ぐきっかけとなりますように。
+          </Text>
+          <Text>皆さまのご来場を心よりお待ちしております。</Text>
+        </Container>
+      </BlurredBox>
+
+      <BlurredBox className={styles.secondGroup}>
+        <Container className={styles.secondHeading}>
+          <Heading as="h2" variant="secondary">
+            第9回赤羽台祭
+          </Heading>
+          <Heading as="h2" variant="secondary">
+            開催概要
+          </Heading>
+        </Container>
+
+        <Container>
+          <dl className={styles.infoSection}>
+            <dt className={styles.infoLabel}>
+              <Text>
+                <Star className={styles.blackStarIcon} />
+                開催日時
+              </Text>
+            </dt>
+            <dd className={styles.infoContent}>
+              <Text>1日目：2025年11月2日(日)10:00～18:00</Text>
+            </dd>
+            <dd className={styles.infoContent}>
+              <Text>2日目：2025年11月3日(月)10:00～18:00</Text>
+            </dd>
+
+            <dt className={styles.infoLabel}>
+              <Text>
+                <Star className={styles.blackStarIcon} />
+                会場
+              </Text>
+            </dt>
+            <dd className={styles.infoContent}>
+              <Text>東洋大学赤羽台キャンパス</Text>
+            </dd>
+            <dd className={styles.infoContent}>
+              <Text>〒115-8650 東京都北区赤羽台1丁目7-11</Text>
+            </dd>
+
+            <dt className={styles.infoLabel}>
+              <Text>
+                <Star className={styles.blackStarIcon} />
+                来場について
+              </Text>
+            </dt>
+            <dd className={styles.infoContent}>
+              <Text>事前予約やチケットについては不要です</Text>
+            </dd>
+            <dd className={styles.infoContent}>
+              <Text>入場料は無料です</Text>
+            </dd>
+            <dd className={styles.infoContent}>
+              <Text>※一部体験や飲食出店は有料です</Text>
+            </dd>
+
+            <dt className={styles.infoLabel}>
+              <Text>
+                <Star className={styles.blackStarIcon} />
+                主催
+              </Text>
+            </dt>
+            <dd className={styles.infoContent}>
+              <Text>赤羽台祭実行委員会</Text>
+            </dd>
+
+            <dt className={styles.infoLabel}>
+              <Text>
+                <Star className={styles.blackStarIcon} />
+                お問い合わせ
+              </Text>
+            </dt>
+            <dd className={styles.infoContent}>
+              <Text>support[at]akabanedai-fes.com</Text>
+            </dd>
+            <dd className={styles.infoContent}>
+              <Text>※ ご連絡の際は[at]を@に変更してお送りください。</Text>
+            </dd>
+          </dl>
+        </Container>
```

Comment:
上で定義した`description`を使うとループで書けます
```suggestion
        <dl className={styles.infoSection}>
          {descriptions.map(({ term, details }) => (
            <Fragment key={term}>
              <Text as="dt" className={styles.infoLabel}>
                <Star className={styles.blackStarIcon} />
                {term}
              </Text>
              {details.map((detail) => (
                <Text as="dd" key={detail} className={styles.infoContent}>
                  {detail}
                </Text>
              ))}
            </Fragment>
          ))}
        </dl>
```